_posts/2020-02-20-contractors.markdown

```diff
@@ -0,0 +1,213 @@
+---
+permalink: v1/contractors
+layout: sidebar
+title: Contractors
+---
+
+# Contractors
+
+## Attributes
+
+| Attribute                     | Type              | Read-Only | Optional | Default | Description
+| :----------                   |:-------------     |:---------:|:--------:|:--------|:-------------
+| `id`                          | Integer           |     X     |          |         | The unique identifier of this contractor in Gusto.
+| `version`                     | String            |     X     |          |         | The version of this object. See <a href="/v1/considerations/versioning">the versioning documentation</a> for a more in depth explanation of versions.
+| `company_id`                  | Integer           |     X     |          |         | The ID of the company to which the contractor belongs.
+| `type`                        | String            |           |          |         | The contractor type, either an "Individual" or a "Business".
+| `start_date`                  | String            |           |          |         | The day when the contractor will start working for the company.
+| `is_active`                   | Boolean           |     X     |          |         | The contractor's status with the company. Contractors created through the API will default to active.
+| `first_name`                  | String            |           |          |         | The contractor's first name. This attribute is required for "Individual" contractors and will be ignored for "Business" contractors.
+| `middle_initial`              | String            |           |    X     | null    | The contractor's middle initial. This attribute is optional for "Individual" contractors and will be ignored for "Business" contractors.
+| `last_name`                   | String            |           |          |         | The contractor's last name. This attribute is required for "Individual" contractors and will be ignored for "Business" contractors.
+| `email`                       | String            |           |    X     | null    | The contractor's email address. This attribute is optional for "Individual" contractors and will be ignored for "Business" contractors. This attribute should only be included in POST requests.
+| `self_onboarding`             | Boolean           |           |    X     | false   | Whether the contractor will complete onboarding in Gusto themselves, or if the administrator should complete their onboarding. It is recommended for contractors to self-onboard so that Gusto accounts are immediately created for them. If `self_onboarding` is true, then `email` is required. This attribute should only be included in POST requests.
```

Comment:
Aside: Interesting that we recommend self onboarding, but we default this to false. Haha. :) I recommend a slightly simpler wording and expansion on what happens outside of a POST:

> Specifies if the contractor or the administrator will complete onboarding in Gusto...If you include `self_onboarding` in a PUT request, it will be ignored.

Reply:
Yeah, there's a tooltip in-app that recommends self-onboarding, but the admin still has the check the box and enter an email which I suppose is why it defaults to false. 

Thanks for the wording improvement!